fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/impl/registry/sync/FabricRegistryInit.java

```diff
@@ -0,0 +1,175 @@
+
+		// Serialised by string, doesnt seem to be synced
+		RegistryAttributeHolder.get(Registry.FOLIAGE_PLACER_TYPE);
+
+		// Serialised by string, doesnt seem to be synced
+		RegistryAttributeHolder.get(Registry.TRUNK_PLACER_TYPE);
+
+		// Serialised by string, doesnt seem to be synced
+		RegistryAttributeHolder.get(Registry.TREE_DECORATOR_TYPE);
+
+		// Serialised by string, doesnt seem to be synced
+		RegistryAttributeHolder.get(Registry.FEATURE_SIZE_TYPE);
+
+		// Synced in ParticleS2CPacket
+		RegistryAttributeHolder.get(Registry.PARTICLE_TYPE)
+				.addAttribute(RegistryAttribute.SYNCED);
+
+		// Serialised by string, doesnt seem to be synced
+		RegistryAttributeHolder.get(Registry.BIOME_SOURCE_TYPE);
+
+		// Serialised by string, doesnt seem to be synced
+		RegistryAttributeHolder.get(Registry.CHUNK_GENERATOR_TYPE);
+
+		// Synced in GameJoinS2CPacket and PlayerRespawnS2CPacket. Entities and maps also use the id when saving.
+		RegistryAttributeHolder.get(Registry.DIMENSION_TYPE)
+				.addAttribute(RegistryAttribute.SYNCED)
+				.addAttribute(RegistryAttribute.PERSISTED);
+
+		// Synced in PaintingSpawnS2CPacket
+		RegistryAttributeHolder.get(Registry.PAINTING_MOTIVE)
+				.addAttribute(RegistryAttribute.SYNCED);
+
+		// Doesnt seem to be synced or saved, STAT_TYPE seems to handle the syncing.
+		RegistryAttributeHolder.get(Registry.CUSTOM_STAT);
+
+		// Serialised by string, doesnt seem to be synced
+		RegistryAttributeHolder.get(Registry.CHUNK_STATUS);
+
+		// Serialised by string, doesnt seem to be synced
+		RegistryAttributeHolder.get(Registry.STRUCTURE_FEATURE);
+
+		// Serialised by string, doesnt seem to be synced
+		RegistryAttributeHolder.get(Registry.STRUCTURE_PIECE);
+
+		// Serialised by string, doesnt seem to be synced
+		RegistryAttributeHolder.get(Registry.RULE_TEST);
+
+		// Serialised by string, doesnt seem to be synced
+		RegistryAttributeHolder.get(Registry.POS_RULE_TEST);
+
+		RegistryAttributeHolder.get(Registry.STRUCTURE_PROCESSOR);
+
+		// Serialised by string, doesnt seem to be synced
+		RegistryAttributeHolder.get(Registry.STRUCTURE_POOL_ELEMENT);
+
+		// Synced in OpenScreenS2CPacket
+		RegistryAttributeHolder.get(Registry.SCREEN_HANDLER)
+				.addAttribute(RegistryAttribute.SYNCED);
+
+		// Does not seem to be serialised, only queried by id. Not synced
+		RegistryAttributeHolder.get(Registry.RECIPE_TYPE);
+
+		// Synced by id
+		RegistryAttributeHolder.get(Registry.RECIPE_SERIALIZER);
+
+		// Synced and saved by id
+		RegistryAttributeHolder.get(Registry.ATTRIBUTES);
+
+		// Synced in StatisticsS2CPacket
+		RegistryAttributeHolder.get(Registry.STAT_TYPE)
+				.addAttribute(RegistryAttribute.SYNCED);
+
+		// Synced by rawID in TrackedDataHandlerRegistry.VILLAGER_DATA
+		RegistryAttributeHolder.get(Registry.VILLAGER_TYPE)
+				.addAttribute(RegistryAttribute.SYNCED);
+
+		// Synced by rawID in TrackedDataHandlerRegistry.VILLAGER_DATA
+		RegistryAttributeHolder.get(Registry.VILLAGER_PROFESSION);
```

Comment:
Need a synced here, you missed that line

Reply:
Yes, thank you.